kong/conf_loader.lua

```diff
@@ -123,12 +143,14 @@ local CONF_INFERENCES = {
     deprecated = {
       replacement = "nginx_http_upstream_keepalive",
       alias = function(conf)
-        -- called before check_and_infer(), must parse ourselves
-        if tonumber(conf.upstream_keepalive) == 0 then
-          conf.nginx_http_upstream_keepalive = "NONE"
+        if conf.upstream_keepalive then
```

Comment:
It seems that this fix to `conf.upstream_keepalive` and `conf.nginx_http_upstream_keepalive` should be on a separate commit, if I'm reading this correctly it has nothing to do with `cassandra_consistency_*`

Reply:
This check is unnecessary, since we have `if conf[property_name] ~= nil` at https://github.com/Kong/kong/blob/next/kong/conf_loader.lua#L822. Besides, it may even harmful since exposing an anti-pattern that could be copy-pasted for those boolean configuration properties.